packages/compiler/src/init/init.ts

```diff
@@ -286,7 +287,10 @@ async function validateTemplate(template: any, templatesUrl: TemplatesUrl): Prom
   const validationTarget = templatesUrl.file as SourceFile;
   let validationResult: ValidationResult;
   // 1. If current version > minimumCompilerVersion, proceed with strict validation
-  if (template.minimumCompilerVersion && currentCompilerVersion > template.minimumCompilerVersion) {
+  if (
+    template.minimumCompilerVersion &&
```

Comment:
if we using `semver` pacakge do we maybe just want to leave that as a semver range(`>=0.45.0`) and rename to `compilerVersion`

Reply:
I like it.

Reply:
I like the rename. Not sure about "leave that" means. `compilerVersion` default value as ">=0.46.0" ?